bom-latest/pom.xml

```diff
@@ -216,6 +227,11 @@
                 <artifactId>pipeline-input-step</artifactId>
                 <version>2.11</version>
             </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.plugins</groupId>
+                <artifactId>pipeline-stage-step</artifactId>
+                <version>2.3</version>
```

Comment:
This release is really old and is missing https://github.com/jenkinsci/jenkins-test-harness/pull/79.